Add section on (lack of) interaction with Storage Access API permissions. (#42)
cwilso/CHIPS: README.md

    This will allow callers of [`CookieStore.delete`](https://wicg.github.io/cookie-store/#CookieStore-delete) to specify whether they want to delete a `partitioned` cookie.
    If the field is not present, it will default to `false`.

### Storage Access API

> Partitioned cookies should be accessible regardless of any choices the user has made via the Storage Access API in the given context. I.e., even if the user has rejected or dismissed a prompt for storage access, partitioned cookies should still be accessible. (Note that partitioned LocalStorage will still be readable/writable in such a context, so blocking access to partitioned cookies would have no effect on user privacy.)

Reviewer: ```suggestion
Partitioned cookies should be accessible regardless of any choices the user has made via the Storage Access API in the given context. Even if the user has rejected or dismissed a prompt for storage access, partitioned cookies should still be accessible (partitioned LocalStorage will still be available in such a context, so blocking access to partitioned cookies would have no effect on user privacy).
```

> Partitioned cookies should be accessible regardless of any choices the user has made via the Storage Access API in the given context. I.e., even if the user has rejected or dismissed a prompt for storage access, partitioned cookies should still be accessible. (Note that partitioned LocalStorage will still be readable/writable in such a context, so blocking access to partitioned cookies would have no effect on user privacy.)

Reviewer: Do you have a reference to somewhere in the storage partitioning repo we mention that it is available regardless of SAAPI status?

Author: No, the storage partitioning proposal does not go into it. However the Storage Access API has had numerous discussions about whether `document.requestStorageAccess` should request storage access (e.g. `localStorage`) as opposed to just cookies access:

* https://github.com/privacycg/storage-access/issues/4
* https://github.com/privacycg/storage-access/issues/31
* https://github.com/privacycg/storage-access/issues/95

I haven't checked the current Safari/Firefox implementations, but the latest comments from those implementors indicate that they are aligning on scoping SAA to be about access to cookies and not related to other kinds of storage. If that's the case, then things like `localStorage` wouldn't be affected by SAA, and would therefore still be accessible, but would be partitioned per the storage partitioning proposal.
